src/main.c

```diff
@@ -27,7 +27,7 @@ int main(int argc, char **argv)
     agent_config* config = (agent_config*) malloc(sizeof(agent_config));
     ALLOC_CHECK(NULL, "Unable to asssign memory for agent config.");
     int config_src_type = argc >= 2 ? READ_FROM_CMD : READ_FROM_FILE;
-    config = read_agent_config(config_src_type, "pcp-statsd-pmda.conf", argc, argv);
+    config = read_agent_config(config_src_type, "config.ini", argc, argv);
```

Comment:
Extension ini already imply configuration data, I'd go with `statsd-pmda.ini` or something like that.

Reply:
Fixed by 45c6bf7